src/components/unique-token/UniqueTokenAttributes.tsx

```diff
@@ -1,64 +1,74 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { sortList } from '../../helpers/sortList';
 import { magicMemo } from '../../utils';
-import Tag from '../Tag';
+import Tag from './Tag';
 import { Inline } from '@rainbow-me/design-system';
 import { UniqueAsset } from '@rainbow-me/entities';
-
-interface AttributeItemProps {
-  color: string;
-  disableMenu?: boolean;
-  trait_type: string;
-  slug: string;
-  value: string | number;
-  max_value?: string | number;
-}
-
-const renderAttributeItem = ({
-  color,
-  disableMenu,
-  trait_type: type,
-  slug,
-  value,
-  max_value: maxValue,
-}: AttributeItemProps) =>
-  type && value ? (
-    <Tag
-      color={color}
-      disableMenu={disableMenu}
-      key={`${type}${value}`}
-      maxValue={maxValue}
-      slug={slug}
-      text={value}
-      title={type}
-    />
-  ) : null;
+import isHttpUrl from '@rainbow-me/helpers/isHttpUrl';
+import transformUniqueAssetTraitsForPresentation from '@rainbow-me/helpers/transformUniqueAssetTraitsForPresentation';
+import uniqueAssetTraitDisplayTypeCompareFunction from '@rainbow-me/helpers/uniqueAssetTraitDisplayTypeCompareFunction';
 
 interface UniqueTokenAttributesProps {
   color: string;
-  disableMenu?: boolean;
+  isPoap?: boolean;
   slug: string;
   traits: UniqueAsset['traits'];
 }
 
 const UniqueTokenAttributes = ({
   color,
-  disableMenu,
+  isPoap,
   slug,
   traits,
 }: UniqueTokenAttributesProps) => {
-  const sortedTraits = (sortList(
-    traits,
-    'trait_type',
-    'asc'
-  ) as typeof traits).map(trait => ({
-    ...trait,
-    color,
-    disableMenu,
-    slug,
-  }));
+  const sortedTraits = useMemo(
+    () =>
+      (sortList(traits, 'trait_type', 'asc') as typeof traits)
+        .filter(
+          trait => trait.value && trait.trait_type && !isHttpUrl(trait.value)
```

Comment:
Since `trait.value` can be a number, the truthiness check will omit any traits where the value is `0`.